Keras_Tensorflow/04_DeployOnAKS.ipynb

```diff
@@ -35,25 +35,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Below are the various name definitions for the resources needed to setup AKS as well as the name of the Docker image we will be using."
+    "Below are the various name definitions for the resources needed to setup AKS as well as the name of the Docker image we will be using. If you wish to use the image that you previously pushed to your account, make sure you change this."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 1,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Please modify the below as you see fit\n",
-    "resource_group = \"<RESOURCE_GROUP>\" \n",
-    "aks_name = \"<AKS_CLUSTER_NAME>\"\n",
+    "resource_group = \"mabouaks\"\n",
+    "aks_name = \"mabouaks\"\n",
     "location = \"eastus\"\n",
     "\n",
-    "image_name = '<YOUR_DOCKER_IMAGE>' # 'fboylu/kerastf-gpu' Feel free to use this image if you want to \n",
-    "                                   # skip creating your own container\n",
-    "selected_subscription = \"'<YOUR_SUBSCRIPTION>'\" # If you have multiple subscriptions select \n",
+    "docker_login = 'mabouatmicrosoft'  # 'fboylu/kerastf-gpu' Feel free to use this image if you want to \n",
+    "image_name = docker_login + '/kerastf-gpu' # skip creating your own container\n",
+    "\n",
+    "selected_subscription = \"'Boston-DS-Brandon-Dev'\" # If you have multiple subscriptions select \n",
```

Comment:
Okay, but use the "docker_login"?

Reply:
yes, I think you keep docker_login and provide your login so that they can run this notebook stand alone if they want. you would need to delete the comment #fboylu/kerastf-gpu next to as well.